.github/ISSUE_TEMPLATE/performance_issue.yml

```diff
@@ -0,0 +1,54 @@
+name: Feature Request
+description: Suggest an idea for vtk.js.
+title: ''
+labels: 'type: feature request 💡'
+assignees:
+body:
+  - type: markdown
+    attributes:
+      value: |
+        👋 Hello, and thank you for starting this contribution!
+
+        ❓ If you are trying to submit a question for general support, please reach out on our discourse forum instead: https://discourse.vtk.org/c/web/9
+
+        🐞 If you do need to report a bug, please follow the template below to increase the chances of your report being looked at.
```

Comment:
This should be removed?    The template is for Performance Issues, not bugs, correct?